ui/test/nuxeo-error.test.js

```diff
@@ -0,0 +1,134 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { fixture, flush, html, isElementVisible } from '@nuxeo/nuxeo-elements/test/test-helpers.js';
+import '../nuxeo-error.js';
+/* eslint-disable no-unused-expressions */
+
+suite('<nuxeo-error>', () => {
+  const expectNuxeoErrorContent = (nuxeoError, code, description, url, message) => {
+    expect(isElementVisible(nuxeoError)).to.be.true;
+
+    const errorCode = nuxeoError.shadowRoot.querySelector('.code');
+    expect(isElementVisible(errorCode)).to.be.true;
+    expect(errorCode.innerText).to.equal(code);
+
+    const errorDescription = nuxeoError.shadowRoot.querySelector('.description');
+    expect(isElementVisible(errorDescription)).to.be.true;
+    expect(errorDescription.innerText).to.equal(description);
+
+    const errorUrl = nuxeoError.shadowRoot.querySelector('.url');
+    expect(isElementVisible(errorUrl)).to.be.true;
+    expect(errorUrl.innerText).to.equal(url);
+
+    const errorMessage = nuxeoError.shadowRoot.querySelector('.message');
+    expect(isElementVisible(errorMessage)).to.be.true;
+    expect(errorMessage.innerText).to.equal(message);
+  };
+
+  const expectNuxeoErrorEmptyContent = (nuxeoError) => {
+    expect(isElementVisible(nuxeoError)).to.be.true;
+
+    const code = nuxeoError.shadowRoot.querySelector('.code');
+    expect(isElementVisible(code)).to.be.false;
+    expect(code.innerText).to.be.empty;
+
+    const description = nuxeoError.shadowRoot.querySelector('.description');
+    expect(isElementVisible(description)).to.be.false;
+    expect(description.innerText).to.be.empty;
+
+    const url = nuxeoError.shadowRoot.querySelector('.url');
+    expect(isElementVisible(url)).to.be.false;
+    expect(url.innerText).to.be.empty;
+
+    const message = nuxeoError.shadowRoot.querySelector('.message');
+    expect(isElementVisible(message)).to.be.false;
+    expect(message.innerText).to.be.empty;
+  };
```

Comment:
nitpick: If you think it is reasonable, these two methods can be merged in one, since the only difference between them `to.equal` vs `to.be.empty` (achievable with if clause or ternary, for example)